content/ja/docs/concepts/workloads/controllers/_index.md

```diff
@@ -1,4 +1,4 @@
 ---
-title: "コントローラー"
+title: "Workload Resources"
```

Comment:
```suggestion
title: "ワークロードリソース"
```
Here is the place for Japanese translation.
So I think that it's good to write in Japanese.

Reply:
done